README.md

```diff
@@ -33,9 +30,9 @@ Show your support
 Give a ⭐️ if you like this project!
 
 Acknowledgments
-Hat tip to anyone whose code was used
-Inspiration
-etc
+original design by : Cindy Chin 
+
+
 License
 This project is MIT licensed.
```

Comment:
- [ ] [OPTIONAL] It will be better if you format your readme with a markdown structure to make your readme has a good scene and be readable to attract recruiters.